### DISCUSSION TITLE
improvement of https://pyodide.org/

### DISCUSSION THREAD

**Discussion by grimmerk** [pyodide/pyodide#1954] [General]
What I think of 
- Put Pyodide GitHub URL (or within GitHub icon) somewhere so people can navigate between GitHub and the official site. 
- [My preference] Put a sentence to tell people how to pronounce `Pyodide`. Actually, I learn it from some Youtube videos.

**Comment by hoodmane** [comment]
Pronunciation in IPA of Pyodide is `ˈpɑɪ.ə.ˌdɑɪd`.

Here's audio:
https://user-images.githubusercontent.com/8739626/141611010-5fead69a-ac7e-48c0-8afd-ca1251cbd7d1.mp4

I'm +1 to include pronunciation guide, we've had several people ask.

**Reply by ryanking13** [reply]
Not important, but I'm also curious of where the name "Pyodide" came from. I know it's from "iodide" but I don't know where "iodide" came from.

**Reply by grimmerk** [reply]
I wonder, too.

**Reply by rth** [reply]
I don't remember. @wlach might know how the "Iodide" name was chosen and if it has a meaning.

**Reply by wlach** [reply]
IIRC iodide was chosen because it sounded good, there was a "scientific" connection and the iodide.io domain was available. 😄 @hamilton may remember better than me or have additional details to share.

**Comment by rth** [comment]
+1 for better inter-linking. We also need to mention the blog somewhere. For the pronunciation maybe we could put it in https://pyodide.org/en/stable/project/about.html ? 

One day it could be good to have a better home page in general (for instance as numpy does https://numpy.org/ which is different from the documentation summary https://numpy.org/doc/stable/) but I'm not sure we could do this with ReadtheDocs, currently there is redirect from `/` to `/en/stable/` and I haven't found a way to define a separate home page.

**Reply by rth** [reply]
I think in general we should spend more documentation effort on pages where users are most likely to end up. For instance below is the monthly visit stats given by readthedocs which show that most visits are for the home page and quickstart. Also most of those come from Google search not Github, and so indeed without the extra context  provided by the GH readme, the current presentation might have a bit too many links for new users.
There are also 20x fewer people visiting "about" page as compared to the home page. So it looks like most relevant information about the project should be on the home page.
```
index.html 6407
usage/quickstart.html 4223
usage/index.html 1539
usage/loading-packages.html 1416
usage/downloading-and-deploying.html 1077
usage/api/js-api.html 988
usage/webworker.html 765
usage/faq.html 742
usage/type-conversions.html 638
usage/api/python-api.html 633
development/new-packages.html 586
usage/api-reference.html 580
development/building-from-sources.html 430
usage/api/micropip-api.html 398
project/related-projects.html 382
project/roadmap.html 370
project/about.html 350 
usage/wasm-constraints.html 481 
```
I'll make a PR to improve the home page somewhat.

**Comment by grimmerk** [comment]
hi, I have another idea to put Pyodide example of https://jsfiddle.net/ in https://pyodide.org/ or/and GitHub, besides the original REPL link.

Pros.: a playground to let users have JS and Python interaction. 
Cons. a user needs to wait for 3,4s to load Pyodide for this, it might be acceptable for most people.

**Reply by rth** [reply]
Yes, we could do that. It's true that users might not necessarily think of opening a JS console to play with Python/JS interaction I'm not sure what would be best between jsfiddle / codepen.io and other similar sites.

**Reply by grimmerk** [reply]
I prefer jsfiddle since it has an embedded console feature. But I do not use them very often so I can not make a good comparision. Or just choose one first and we can change it later if we find out a better one.
